README.md

```diff
@@ -126,7 +126,17 @@ Users can create CloudFirewall instances, supply their own rules and attach them
 **Note**<br/>
 If the user supplies a firewall-id, and later switches to using an ACL, the CCM will take over the CloudFirewall Instance. To avoid this, delete the service, and re-create it so the original CloudFirewall is left undisturbed.
 
+#### Routes
+When running k8s clusters within VPC, node specific podCIDRs need to be allowed on the VPC interface. Linode CCM comes with route-controller functionality which can be enabled for automatically adding/deleting routes on VPC interfaces. When installing CCM with helm, make sure to specify routeController settings.
 
+##### Example usage in values.yaml
+```yaml
+routeController:
+  vpcName: <name of VPC>
+  linodeNodePrivateSubnet: 10.0.0.0/8
+  configureCloudRoutes: true
+  routeReconciliationPeriod: 1m
```

Comment:
is this routeReconciliationPeriod still used? can't immediately find where it is used in the code

Reply:
This is configured by [cloud-provider](https://github.com/linode/linode-cloud-controller-manager/blob/main/go.mod#L16C1-L17C1) package by default.
```
k logs daemonset/ccm-linode -n kube-system | grep route-recon
Found 3 pods, using pod/ccm-linode-gmwt5
I0311 21:19:47.416740       1 flags.go:64] FLAG: --route-reconciliation-period="1m0s"
```
If not set, the default is 10s.
https://github.com/kubernetes/cloud-provider/blob/master/config/v1alpha1/defaults.go#L69